ASP_tests.py

```diff
@@ -31,22 +38,89 @@ def test_SNID_to_loc():
     assert host_ra == 7.731832
     assert host_dec == -44.459011
 
+
 def test_findAllExposures():
     explist = findAllExposures(50134575, 7.731890048839705, -44.4589649005717,62654.,62958.,62683.98, 'Y106', maxbg = 24, maxdet = 24, \
                         return_list = True, stampsize = 25, roman_path = roman_path,\
                     pointing_list = None, SCA_list = None, truth = 'simple_model')
-    compare_table = ascii.read('tests/testdata/findallexposurestest.dat') 
+    compare_table = ascii.read('tests/testdata/findallexposurestest.dat')
     assert explist['Pointing'].all() == compare_table['Pointing'].all()
     assert explist['SCA'].all() == compare_table['SCA'].all()
     assert explist['date'].all() == compare_table['date'].all()
 
-def test_simulateImages():
-    images, im_wcs_list, cutout_wcs_list = simulateImages(10,5,7.541534306163982, -44.219205940734625, True, True, \
-        [10, 100, 1000, 10**4, 10**5], 0, False, 'F184', size=11)
+
+def test_simulate_images():
+    band = 'F184'
+    lam = 1293  # nm
+    airy = \
+        galsim.ChromaticOpticalPSF(lam, diam=2.36, aberrations=galsim.roman.
+                                   getPSF(1, band, pupil_bin=1).aberrations)
+    images, im_wcs_list, cutout_wcs_list, psf_storage, sn_storage = \
+        simulate_images(testnum=10, detim=5, ra=7.541534306163982,
+                        dec=-44.219205940734625, do_xshift=True,
+                        do_rotation=True, supernova=[10, 100, 1000,
+                                                     10**4, 10**5],
+                        noise=0, use_roman=False, band='F184',
+                        deltafcn_profile=False, roman_path=roman_path, size=11,
+                        input_psf=airy, bg_gal_flux=9e5)
     compare_images = np.load('tests/testdata/images.npy')
     assert compare_images.all() == images.all()
 
 
+def test_simulate_wcs():
+    wcs_dict = simulate_wcs(angle=np.pi/4, x_shift=0.1, y_shift=0,
+                            roman_path=roman_path, base_sca=11,
+                            base_pointing=662, band='F184')
+    b = np.load('./tests/testdata/wcs_dict.npz', allow_pickle=True)
+    assert wcs_dict == b, 'WCS simulation does not match test example'
+
+
+def test_simulate_galaxy():
+    band = 'F184'
+    roman_bandpasses = galsim.roman.getBandpasses()
+    lam = 1293  # nm
+    sed = galsim.SED(galsim.LookupTable([100, 2600], [1, 1],
+                     interpolant='linear'), wave_type='nm',
+                     flux_type='fphotons')
+    sim_psf = \
+        galsim.ChromaticOpticalPSF(lam, diam=2.36, aberrations=galsim.roman.
+                                   getPSF(1, band, pupil_bin=1).aberrations)
+    convolved = simulate_galaxy(bg_gal_flux=9e5, deltafcn_profile=False,
```

Comment:
Is there a random seed you need to control here to make sure the noise realization is the same for `simulate_galaxy` and `simulate_supernovae`?